README.md

```diff
@@ -314,6 +314,8 @@ Table of Contents
   * [Shieldfy](https://shieldfy.io) — Web application firewall and vulnerability detection for developers, free plan up to 100k requests per month.
   * [Sqreen](https://www.sqreen.com/) — Application security monitoring and protection (RASP, WAF and more) for web applications and APIs. Free for 1 app and 3 million requests.
   * [Sucuri SiteCheck](https://sitecheck.sucuri.net) - Free website security check and malware scanner
+  * [Is it Hacked](http://isithacked.com) - Free cloud website security check and malware scanner. Email notifications for registered users.
+  * [Protectumus](https://protectumus.com) - Free website security check, site antivirus and server firewall (WAF). Email and SMS notifications for registered users.
```

Comment:
The WAF isnt included in the free tier.

Reply:
The WAF is included in the free tier. The free plan also provides: "Infections Scan", "Malware detection" and "Bot protection" which are part of the WAF. The paid plans offer more features, but the free plan offers basi WAF functions, which were mentioned above. Thanks for checking the website.